mainnet/jobs.json

```diff
@@ -78,5 +78,69 @@
     "name": "ethusd_cryptocompare",
     "selector": "[USD]",
     "url": "https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_gemini",
+    "selector": "last",
+    "url": "https://api.gemini.com/v1/pubticker/btcusd"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_kucoin",
+    "selector": "data.BTC",
+    "url": "https://api.kucoin.com/api/v1/prices?base=USD&currencies=BTC"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_kraken",
+    "selector": "result.XXBTZUSD.c[0]",
+    "url": "https://api.kraken.com/0/public/Ticker?pair=BTCUSD"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "btcusd_binance",
+    "selector": "price",
+    "url": "https://api.binance.com/api/v3/avgPrice?symbol=BTCBUSD"
```

Comment:
hey @super1114, only pure BTC/USD pairs will be accepted, no stable coins or wrapped pairs. This is a security measure. Thank you!

Reply:
nudge to make this change @super1114